package.json

```diff
@@ -25,11 +25,11 @@
   },
   "homepage": "https://github.com/devongovett/browserify-istanbul",
   "dependencies": {
-    "istanbul": "~0.3.6",
     "minimatch": "^2.0.0",
     "through": "^2.3.4"
   },
   "devDependencies": {
+    "istanbul": "^0.4.2",
```

Comment:
It needs `istanbul` to be present to run,
and in dev environment (outside of an actual app)
it's the only source.